starter-code/src/viking.js

```diff
@@ -1,11 +1,72 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength =  strength;
+}
+Soldier.prototype.attack =  function() {
+return this.strength;
+}
+Soldier.prototype.receiveDamage = function(damage) {
+
+this.health = this.health - damage;
+console.log(this.health);
+}
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength) {
+  this.name = name; 
```

Comment:
Aqui deben utilizar la herencia de propiedades. Solucion:
function Viking(name, health, strength) {
    this.name = name;
    Soldier.call(this, health, strength);
}